### ISSUE TITLE
Page jumps around when scrolling down after tiles have loaded

### ISSUE THREAD

**Issue by tomgco** [numo-labs/isearch-ui#222]
Noticeable on the homepage.

**Comment by tomgco** [comment]
This seems to happen after a console.log:
```
----RENDERING GRID----
action @ 16:38:47.747 UPDATE_DISPLAYED_ITEMS 
```

**Comment by nikhilaravi** [comment]
This should have been fixed after #224. Please check if it is still an issue and let me know!
